src/components/LoggedIn.tsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+
+export const LoggedIn = () => {
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  
+  const handleLogin = () => {
+    setIsLoggedIn(true);
+    console.log(isLoggedIn);
+  }
```

Comment:
Can you remove these `console.log` please?

Reply:
got it.